microsoft/PowerToys
# Copy Indexer Languages Folder during instalation

Do to recent changes in master Powertoys run Indexer needs languages files that are not being copied by the installer.
Powertoys run was writing a lot of logs (because of the lack of language files) every time a letter has written which made antimalware Service Executable spike for some reason.

## Validation Steps Performed
Manual Verification by building the installer, install it and verifying that the languages folder of the indexer plugin were there.
Open Powertoys write anything and see antimalware Service Executable running normal.
